installation/deploy-syskit-point.md

```diff
@@ -16,9 +16,10 @@ Prepare the following for SysKit Point deployment:
 ## Azure Marketplace
 
 * start by opening the [SysKit Point product page](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/syskitltd.syskit_point) on Azure Marketplace
-* click the __Get It Now button (1)__; a confirmation dialog opens
-* click the __Continue (2) button__; you will be redirected to Azure Portal
-* click __Create (3)__ to continue with the deployment configuration; __Create SysKit Point__ screen opens with the __Basics__ tab selected
+* Click the __Get It Now button (1)__; a confirmation dialog opens
+* Click the __Continue (2) button__; you will be redirected to Azure Portal
+* Select the **Plan (3)** from the dropdown menu
+* Click __Create (4)__ to continue with the deployment configuration; __Create SysKit Point__ screen opens with the __Basics__ tab selected
 
 ![SysKit Point on Azure Marketplace](../.gitbook/assets/deploy-syskit-point_get.png)
 ![Create App in Azure](../.gitbook/assets/deploy-syskit-point_continue.png)
```

Comment:
treba uzeti novu sliku za ![SysKit Point in Azure](../.gitbook/assets/deploy-syskit-point_create.png)
mora imati vidljiv onaj Plan dropdown

Reply:
https://sharexuploadsstorage.blob.core.windows.net/sharexuploads/ShareX/2022/09/IMESARIC_02.09.2022_08.26.46.png